src/compiler/statement/IfStatement.ts

```diff
@@ -0,0 +1,38 @@
+import * as ts from "typescript";
+import {removeStatementedNodeChild} from "./../../manipulation";
+import {Node, Expression} from "./../common";
+import {ChildOrderableNode} from "./../base";
+import {Statement} from "./Statement";
+
+export const IfStatementBase = ChildOrderableNode(Statement);
+export class IfStatement extends IfStatementBase<ts.IfStatement> {
+    /**
+     * Gets this if statement's expression.
+     */
+    getExpression() {
+        return this.global.compilerFactory.getNodeFromCompilerNode(this.compilerNode.expression, this.sourceFile) as Expression;
+    }
+
+    /**
+     * Gets this if statement's then statement.
+     */
+    getThenStatement() {
+        return this.global.compilerFactory.getNodeFromCompilerNode(this.compilerNode.thenStatement, this.sourceFile) as Statement;
+    }
+
+    /**
+     * Gets this if statement's else statement.
+     */
+    getElseStatement() {
+        return this.compilerNode.elseStatement == null
+            ? null
```

Comment:
Keep null for the check above, but return `undefined` here. The public api of the library only uses `undefined`.